runtime/exporter-stdout/src/main/java/io/aklivity/zilla/runtime/exporter/stdout/internal/StdoutExporterContext.java

```diff
@@ -48,6 +51,11 @@ public ExporterHandler attach(
         Collector collector,
         LongFunction<KindConfig> resolveKind)
     {
+        if (!attached.compareAndSet(false, true))
+        {
+            return NOOP_HANDLER;
+        }
+
```

Comment:
This doesn't seem like it would work for the scenario we discussed, where there are 2 namespaces, both with stdout exporter, and the first one is detached, then the second one should still produce output.

However, this approach makes the second one a no-op, so the second one doesn't produce any output even after the first namespace is detached.

To make this work, we need to short circuit in the handler itself, not in attach, so the handler can take over when needed.